src/index.html

```diff
@@ -15,8 +15,79 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="origin"
```

Comment:
The 'crossorigin' attribute should have the value 'anonymous' instead of 'origin'. This is a common value used when preconnecting to external resources like Google Fonts.